google_auth_oauthlib/flow.py

```diff
@@ -379,6 +379,7 @@ def run_local_server(
         redirect_uri_trailing_slash=True,
         timeout_seconds=None,
         token_audience=None,
+        browser=None,
```

Comment:
Please add this parameter to the doc string

Reply:
@clundin25 I have added the doc string